CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Fix: Multiple FlutterError.onError calls in FlutterErrorIntegration (#345)
 * Fix: Pass hint to EventProcessors (#356)
 * Fix: EventProcessors were not dropping events when returning null (#353)
+* Fix: Prefix App, Browser, Device, Message, Request and User with Sentry (#250)
```

Comment:
```suggestion

Breaking Changes:

* Fix: Prefix App, Browser, Device, Message, Request and User with Sentry (#250)

// let's describe here the renamed Protocols and that now they have a 'Sentry' prefix.
```

Reply:
we'll probably need to add a Migration page to https://docs.sentry.io/platforms/dart/ but let's track this in another issue :)